### ISSUE TITLE
Accessibility [a11y] - JAWS cannot read the full EnablementDialog

### ISSUE THREAD

**Issue by GeraldMit** [eclipse-rap-incubator/org.eclipse.rap.incubator.e4.compatibility.workbench-2#4]
JAWS likely doesn't read it out because labels are supposed to be labels for fields according to JAWS so it only reads out the first one....

https://github.com/eclipse/org.eclipse.rap.incubator.e4.compatibility.workbench/blob/master/bundles/org.eclipse.rap.ui.workbench/src/org/eclipse/ui/internal/activities/ws/EnablementDialog.java

looking at the code, line 117 is where a second Label is created right after the first
```
91	protected Control createDialogArea(Composite parent) 
...
117             text = new Label(composite, SWT.NONE);
118			text
119					.setText(strings
120							.getProperty(
121									WorkbenchTriggerPointAdvisor.PROCEED_SINGLE,
122									RESOURCE_BUNDLE
123											.getString(WorkbenchTriggerPointAdvisor.PROCEED_SINGLE)));

```

This means that line 156 also doesn't work with JAWS

```
156            text = new Label(composite, SWT.NONE);
157			text
158					.setText(strings
159							.getProperty(
160									WorkbenchTriggerPointAdvisor.PROCEED_SINGLE,
161									RESOURCE_BUNDLE
162											.getString(WorkbenchTriggerPointAdvisor.PROCEED_SINGLE)));
```
